zerver/openapi/openapi.py

```diff
@@ -13,21 +14,63 @@
 EXCLUDE_PROPERTIES = {
     '/register': {
         'post': {
-            '200': ['max_message_id', 'realm_emoji'],
+            '200': ['max_message_id', 'realm_emoji', 'pointer'],
         },
     },
+    '/settings/notifications': {
+        'patch': {
+            # Extraneous key
+            '200': ['notification_sound', 'enable_login_emails',
+                    'enable_stream_desktop_notifications', 'wildcard_mentions_notify',
+                    'pm_content_in_desktop_notifications', 'desktop_icon_count_display',
+                    'realm_name_in_notifications', 'presence_enabled'],
+        },
+    },
+
     '/zulip-outgoing-webhook': {
         'post': {
             '200': ['result', 'msg', 'message'],
         },
     },
+    '/users/me': {
+        'get': {
+            # Extraneous key
+            '200': ['delivery_email'],
+        },
+    },
+    '/users/{user_id}': {
+        'get': {
+            # Extraneous key
+            '200': ['delivery_email'],
+        },
+        'delete': {
+            # Extraneous key
+            '200': ['delivery_email'],
+        }
+    },
```

Comment:
These are a great find!  We'll want to document these as only sent in organizations with email_address_visiblity (https://zulipchat.com/help/restrict-visibility-of-email-addresses) configured.  But that can be a follow-up PR.

BTW I'd change the comments from "Extraneous key" to "Some responses contain undocumented keys" :)

Reply:
Done